plasma.desktop as Type=XSession set, which is not a supported one.
Lintian warns about it, and other XSession entries have Type=Application.
Adjust the plasma.desktop source file.
---
 login-sessions/plasma.desktop.cmake |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/login-sessions/plasma.desktop.cmake
+++ b/login-sessions/plasma.desktop.cmake
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Type=XSession
+Type=Application
 Exec=${CMAKE_INSTALL_FULL_BINDIR}/startplasma-x11
 TryExec=${CMAKE_INSTALL_FULL_BINDIR}/startplasma-x11
 DesktopNames=KDE
